### ISSUE TITLE
[menu-button] Missing `aria-label` attributes on icon only stories

### ISSUE THREAD

**Issue by jpzwarte** [sl-design-system/components#1511]
### Provide a general summary of the issue here

See title

### 🤔 Expected Behavior?

-

### 😯 Current Behavior

-

### 💁 Possible Solution

- Add a `label` property to `<sl-menu-button>` and apply that as `aria-label` on the nested button
- Log a warning when the button doesn't have a label or aria-label property

### 🖥️ Steps to Reproduce

-

### What browsers are you seeing the problem on? (only for bugs in code)

Chrome

### What operating system are you using?

macOS

### 👤 Your name

Jeroen Zwartepoorte

### 🧢 Your product/team

SLDS

### 🎨 Your Theme(s)

Sanoma Learning
